backend/fleet_management/serializers.py

```diff
@@ -41,4 +47,30 @@ class Meta:
             'id',
             'driver', 'car', 'passengers',
             'date', 'start_mileage', 'end_mileage', 'description',
+            'start_location', 'end_location',
         )
+
+    def create(self, validated_data):
+        # tracks_data = validated_data.pop('tracks')
+        passengers_data = validated_data.pop('passengers')
+        car_data = validated_data.pop('car')
+        car = Car.objects.get(pk=car_data['id'])
+        passengers = Passenger.objects.filter(
+            id__in=[p['id'] for p in passengers_data],
```

Comment:
instead of `p` I'll suggest to pass `passanger_data` is much more readable than `p`